packages/amazonq/src/lsp/config.ts

```diff
@@ -11,7 +11,7 @@ export interface ExtendedAmazonQLSPConfig extends LspConfig {
 }
 
 export const defaultAmazonQLspConfig: ExtendedAmazonQLSPConfig = {
-    manifestUrl: 'https://aws-toolkit-language-servers.amazonaws.com/codewhisperer/0/manifest.json',
+    manifestUrl: 'https://d3akiidp1wvqyg.cloudfront.net/qAgenticChatServer/0/manifest.json',
     supportedVersions: '^3.1.1',
```

Comment:
I believe the `supportedVersions` field should be changed to match the version served by the manifest. `^0.1.0` appears to work for me, (except for the EPIPE issue).